### DISCUSSION TITLE
How do I update the package?

### DISCUSSION THREAD

**Discussion by bgdnr** [firebase/firebase-ios-sdk#7196] [Q&A] [answered]
Hey guys,

I have 6.34 added through Swift Package Manager.
How can I update it to the latest version (v7)? Do I need to remove the package and add the new one?

**Comment by paulb777** [comment, marked as answer]
@bgdnr See the instructions at https://github.com/firebase/firebase-ios-sdk/blob/master/SwiftPackageManager.md
